### DISCUSSION TITLE
jk keybinding may cause input lag when navigating with j/k in neovim terminal

### DISCUSSION THREAD

**Discussion by JohnWickzzz** [ayamir/nvimdots#1117] [General]
I use the lazygit in neovim floating terminal, when I enter j/k move up and down, I feel the operation is very laggy.
I thought it was neovim terminal performance issue, until I found this issue https://github.com/akinsho/toggleterm.nvim/issues/63

After commenting this code
![image](https://github.com/ayamir/nvimdots/assets/12469651/94222c4b-3e14-4be7-89d7-27c854a2260e)
It feels like the operation is finally smooth.

I don't know if others encountered this problem. Just sharing what I learned.

**Comment by Jint-lzxy** [comment]
I think #1119 should fix this up lol

For those who come later, u might wanna check out `:h ttimeout`:
```console
						*'ttimeout'* *'nottimeout'*
'ttimeout'		boolean (default on)
			global
	This option and 'ttimeoutlen' determine the behavior when part of a
	key code sequence has been received by the |TUI|.

	For example if <Esc> (the \x1b byte) is received and 'ttimeout' is
	set, Nvim waits 'ttimeoutlen' milliseconds for the terminal to
	complete a key code sequence. If no input arrives before the timeout,
	a single <Esc> is assumed. Many TUI cursor key codes start with <Esc>.

	On very slow systems this may fail, causing cursor keys not to work
	sometimes.  If you discover this problem you can ":set ttimeoutlen=9999".
	Nvim will wait for the next character to arrive after an <Esc>.
```
